### PR TITLE
Fixes #31441 -- Fixed admin TabularInline problems

### PR DESCRIPTION
**Pull request by hramezani**
Fixes the first two issues of [Ticket 31441](https://code.djangoproject.com/ticket/31441#comment:3).

### CODE REVIEW

**Comment by hramezani** [comment]
How can we run `Selenium` tests :thinking: ?

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Comment by hramezani** [comment]
it seems the broken test isn't related to the patch.

**Comment by carltongibson** [comment]
> it seems the broken test isn't related to the patch.

Ah, yes. That one. 🤨 Yes. That's happens. 

buildbot, test on selenium.

**Comment by carltongibson** [comment]
buildbot, test on selenium.

File: django/contrib/admin/static/admin/css/base.css [hunk 1]
```diff
@@ -623,7 +623,8 @@ td ul.errorlist li {
     padding-left: 0;
 }
 
-.errors input, .errors select, .errors textarea {
+.errors input, .errors select, .errors textarea,
+td ul.errorlist + input, select, textarea {
```
Comment (carltongibson) [inline django/contrib/admin/static/admin/css/base.css:627]: @hramezani Can you verify these selectors please? 

Aren't the last two `, select, textarea` unscoped?

**Comment by hramezani** [comment]
buildbot, test on selenium.

File: django/contrib/admin/static/admin/css/base.css (hunk 1 above)
Comment (hramezani) [reply django/contrib/admin/static/admin/css/base.css]: Ah!, Thanks @carltongibson .
I've changed the selector and also changed the test to check before/after clicking on the save button. Also, I've used `ccs selector` in the test to verify the above selectors.

Comment (carltongibson) [reply django/contrib/admin/static/admin/css/base.css]: Super. Sounds good. I shall take another look.

**Comment by carltongibson** [comment]
buildbot, test on selenium.

**Comment by hramezani** [comment]
Hi @carltongibson, Thanks for your review.
Have you checked the third problem which I described in [this comment](https://code.djangoproject.com/ticket/31441#comment:3)

> admin.TabularInline and admin.StackedInline will not show Hide link when there is an error and formset is expanded

If you agree that this is a problem I can work on it.

**Comment by carltongibson** [comment]
@hramezani: I missed that thanks. Replied on ticket.

**Comment by carltongibson** [comment]
OK, [Windows build passed](https://djangoci.com/job/pull-requests-windows/5226/). Not sure why it's not reporting back. 

And look, even the selenium tests worked this time 💃 :)

**Pull request merged**